node/src/blockchain/blockchain_interface_utils.rs

```diff
@@ -637,6 +639,13 @@ mod tests {
     #[test]
     fn send_payables_within_batch_works() {
         let accounts = vec![make_payable_account(1), make_payable_account(2)];
+        let port = find_free_port();
+        let _blockchain_client_server = MBCSBuilder::new(port)
+            .begin_batch()
+            .response("rpc_result".to_string(), 7)
+            .response("rpc_result_2".to_string(), 8)
```

Comment:
Think about this both @Syther007  and @utkarshg6. Let me ask what you think. Imagine we just have sent our Json with the signed transactions in it, now we are waiting for the response and what we get back from the server is an error saying that we are rate-limited? How do you think our code would react upon its arrival? 
I'm a little worried, if we ignore the content of the response (Which we do because this test is a proof. We send in two garbage responses but the procedure ends up in two successes.), then we could also not to recognize that we're not receiving an OK, but something else. Can you please look into it carefully for me and disprove my hypothesis? Thanks.   
(Edit: Oh, I saw your telling apart `.response()` from `.err_response()`. It makes sense to me now. The problem is, it actually doesn't if you don't know about the specific ergonomics of your api for the MBCS. I can see how you could improve it easily. Just please modify it so that it says `.ok_response()` which will make it symmetrical to `.err_response`).

If all is fine, you also may want to use a bit more expressive strings:

Instead of 
```
      .response("rpc_result".to_string(), 7)
      .response("rpc_result_2".to_string(), 8)
```
rather
```
      .response("irrelevant_ok_rpc_response".to_string(), 7)
      .response("irrelevant_ok_rpc_response_2".to_string(), 8)
```